src/Traits/HasOutputTo.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Worksome\Number\Traits;
+
+use Worksome\Number\Number;
+use Worksome\Number\Parser;
+
+/**
+ * @mixin Number
+ */
+trait HasOutputTo
+{
+    /**
+     * Cast the number object to string.
+     */
+    public function __toString(): string
+    {
+        return $this->toString();
+    }
+
+    /**
+     * Get the string value of this number
+     */
+    public function toString(): string
+    {
+        return $this->value->value;
+    }
+
+    /**
+     * Get the integer value of this number
+     */
+    public function toInteger(): int
+    {
+        return (int) $this->value->value;
+    }
+
+    /**
+     * Get the float value of this number
+     */
+    public function toFloat(): float
+    {
+        return (float) $this->value->value;
+    }
+
+    /**
+     * Get the Roman Numerals equivalent of this value.
+     *
+     * Not super great for values far beyond 1 million. Romans rarely used
+     * numerals for depicting such large numbers -- apparently often used
+     * words instead.
+     */
+    public function toRomanNumerals(): string
```

Comment:
Not 100% sure about including this one. 🤔

Reply:
Fair 👍